src/app/lib/config/services/persian.js

```diff
@@ -22,11 +22,11 @@ const persian = {
     defaultImageAltText: 'BBC News فارسی',
     dir: 'rtl',
     externalLinkText: '، لینک خارجی',
-    imageCaptionOffscreenText: '،توضیح تصویر',
-    videoCaptionOffscreenText: '،توضیح ویدئو',
-    audioCaptionOffscreenText: '،توضیح صدا',
-    defaultCaptionOffscreenText: '،توضیح',
-    imageCopyrightOffscreenText: '،منبع تصویر',
+    imageCaptionOffscreenText: 'توضیح تصویر،',
```

Comment:
Not sure if the commas should be on the right or left.. @sareh could you confirm? :)

Reply:
The spreadsheet has the commas on the left

Reply:
It should look like this in the article:
<img width="816" alt="Screenshot Persian image caption comma" src="https://user-images.githubusercontent.com/3028997/65315021-466f0d00-db8f-11e9-8221-2f953e567fd0.png">
e.g. https://www.test.bbc.com/persian/articles/c4vlle3q337o

Reply:
Sometimes software like spreadsheets shows right-to-left text in another order, because the fields don't have a`dir="rtl"` or equivalent.

Reply:
I've reverted these back to how they were before as that is correct.